### PR TITLE
Update azure.md

### PR DESCRIPTION
**Pull request by theletterf**
Adding the default endpoint for trace ingestion, plus some minor edits.

### CODE REVIEW

**Review by pellared — APPROVED** [review]

File: docs/azure.md [hunk 1]
```diff
@@ -2,18 +2,15 @@
...
-2. Navigate to `Development Tools` > `Extensions`.
-3. Find and install the `SignalFx .NET Tracing` extension.
-4. Navigate to `Settings` > `Configuration`.
-5. Add `New application setting`s to configure the receiver:
+1. Choose your app in Azure App Service.
+2. Go to **Development Tools > Extensions**.
+3. Find and install the **SignalFx .NET Tracing** extension.
+4. Go to **Settings > Configuration**.
+5. Click **New application setting** to add the following settings:
+   * Name: SIGNALFX_ACCESS_TOKEN
```
Comment (pellared) [inline docs/azure.md:10]:
```suggestion
   * Name: `SIGNALFX_ACCESS_TOKEN`
```

File: docs/azure.md [hunk 2]
```diff
@@ -2,18 +2,15 @@
...
-4. Navigate to `Settings` > `Configuration`.
-5. Add `New application setting`s to configure the receiver:
+1. Choose your app in Azure App Service.
+2. Go to **Development Tools > Extensions**.
+3. Find and install the **SignalFx .NET Tracing** extension.
+4. Go to **Settings > Configuration**.
+5. Click **New application setting** to add the following settings:
+   * Name: SIGNALFX_ACCESS_TOKEN 
+   * Value: `[splunk-observability-cloud-access-token]`
+   * Name: SIGNALFX_ENDPOINT_URL
```
Comment (pellared) [inline docs/azure.md:12]:
```suggestion
   * Name: `SIGNALFX_ENDPOINT_URL`
```

**Review by pellared — APPROVED** [review]
nit comments

**Review by Kielek — APPROVED** [review]

File: docs/azure.md [hunk 3]
```diff
@@ -2,18 +2,15 @@
...
-    Name: SIGNALFX_ACCESS_TOKEN 
-    Value: (Your SIGNALFX access token)
-
-    Name: SIGNALFX_ENDPOINT_URL
-    Value: (Your Collector or SignalFX ingest endpoint)
-    ```
-    (See [advanced-config.md](advanced-config.md) for more options.)
-6. Restart the App Service.
\ No newline at end of file
+> **Tip:** To reduce latency and benefit from OTel Collector features, you can set the endpoint URL setting to a Collector instance running in Azure VM over an Azure private network.
```
Comment (RassK) [inline docs/azure.md:16]: Seems the offical term is Azure VNet > https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview

I would like somebody to confirm it, as there might be more optimized and better services. eg: just running the Collector in a container environment, but VNet should be still helpful.

Comment (RassK) [reply docs/azure.md:16]: @pjanotti any ideas?

**Pull request closed without merging**